lib/member.rb

```diff
@@ -0,0 +1,11 @@
+require 'scraped'
+
+class Member < Scraped::HTML
+  field :section do
+    @section ||= MemberSection.new(response: response, noko: noko)
```

Comment:
Passing `noko: noko` is redundant as the `MemberSection` will automatically create an identical `noko` object from the response you're passing in. So you can drop that parameter.